test/BaselineOutput/SingleDebug/Rff/featurized.tsv

```diff
@@ -6,7 +6,7 @@
 #@   col=RffVectorFloat:R4:9-14
 #@ }
 15	8:Label
-5	1	1	1	2	1	3	1	0	0.3418174	-0.465289354	0.247575819	-0.5215741	0.576975167	-0.02080728
-5	4	4	5	7	10	3	2	0	0.566368	-0.112074189	0.169326738	-0.5519618	0.517083168	-0.2568235
-3	1	1	1	2	2	3	1	0	0.497458071	-0.293033749	0.419352561	-0.39683342	0.572824	-0.07215268
-6	8	8	1	3	4	3	7	0	-0.382303447	-0.432640046	0.289225727	-0.4996817	0.571184337	-0.08415316
+5	1	1	1	2	1	3	1	0	-0.157029659	-0.555585265	0.490177631	-0.305056125	0.35670203	-0.453979075
```

Comment:
Note that because RFF as its name suggests uses PRNG extensively, and the random initialization behavior differs between `ConsoleEnvironment` and `MLContext`, that this is why this baseline changed, in case anyone was curious.